apps/site/assets/ts/schedule/components/schedule-finder/ServiceSelector.tsx

```diff
@@ -182,11 +165,20 @@ export const ServiceSelector = ({
               return (
                 <optgroup key={group} label={optGroupTitles[group]}>
                   {servicesByOptGroup[group].map((service: ServiceByOptGroup) =>
-                    serviceDescription(
-                      service.service,
-                      group,
-                      service.servicePeriod,
-                      multipleWeekdays
+                    group === "holiday" ? (
+                      <HolidayServiceOption
```

Comment:
Is there a particular reason you chose to have `<HolidayServiceOption>` and `<ServiceOption>` be two separate components, instead of handling this `group === "holiday"` condition inside a consolidated `<ServiceOption>`?

Reply:
It's basically because the logic for `HolidayServiceOption` is simple and that for `ServiceOption` is complicated, and also because when we call out storm service I'm contemplating having a third `StormServiceOption`. In the near future we'll be simplifying `ServiceOption` as well and perhaps at that point it'll make sense to combine the three together.